workshop/content/intermediate/operations/stack-policy-and-prevention-controls/_index.md

```diff
@@ -0,0 +1,207 @@
+
+
+1. Navigate to the [Amazon SNS Console](https://console.aws.amazon.com/sns/), and choose **Topics**.
+2. You will observe the topic `Topic-2` you created in the stack is still present, and was not deleted during stack deletion.
+
+Congratulations! You have now learned how to define a `DeletionPolicy` resource attribute on a resource to preserve it during stack deletion. For more information, see [`DeletionPolicy` attribute](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html) and [`DeletionPolicy` options](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html#aws-attribute-deletionpolicy-options).
+
+### Challenge
+
+You have now learned to create a stack policy to deny updates to a resource based on a [Logical Resource ID](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html#resources-section-structure-resource-fields). In this exercise, you are tasked with creating a stack policy that denies updates to only resources with a specific resource type. You will create a stack policy to deny updates to the `AWS::RDS::DBInstance` resource type.
+
+{{%expand "Need a hint" %}}
+
+Make use of the [Condition](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/protect-stack-resources.html#stack-policy-reference) key to define `ResourceType`.
+
+{{% /expand %}}
+
+{{%expand "Want to see the solution?" %}}
+
+```
+{
+  "Statement" : [
+    {
+      "Effect" : "Deny",
+      "Principal" : "*",
+      "Action" : "Update:*",
+      "Resource" : "*",
+      "Condition" : {
+        "StringEquals" : {
+          "ResourceType" : ["AWS::RDS::DBInstance"]
+        }
+      }
+    },
+    {
+      "Effect" : "Allow",
+      "Principal" : "*",
+      "Action" : "Update:*",
+      "Resource" : "*"
+    }
+  ]
+}
+```
+
+{{% /expand %}}
+
+Great work! You have now learned how to create a stack policy to deny updates only for a given resource type.
+
+### Cleanup
+
+Choose to follow steps shown next to cleanup resources you created with this lab:
+
+1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/).
+2. Select the stack named `stack-policy-lab` and choose **Delete**.
+3. In the message window, select **Edit termination protection**, and select **Disabled** for **Termination protection**. Choose **Save**.
+4. Select the stack named `stack-policy-lab` and choose **Delete**, and then choose **Delete stack** to confirm.
+5. Navigate to the [Amazon SNS Console](https://console.aws.amazon.com/sns/), and choose **Topics**. Next, select the topic `Topic-2`, and choose **Delete**. In the message pane, enter `delete me`, and choose **Delete** to confirm.
+
+* * *
+
+### Conclusion
+
+Congratulations! You have learned how to prevent unintentional updates, protect stack from deletion, and preserve resources in case of stack deletion.
```

Comment:
```suggestion
Congratulations! You have learned how to prevent unintentional updates, protect a stack from deletion, and preserve resources in case of an unintentional stack deletion.
```